ProcessMaker/Http/Controllers/Api/Administration/RolesController.php

```diff
@@ -79,4 +79,24 @@ public function create(Request $request)
         return fractal($role, new RoleTransformer())->respond();
     }
 
+    /**
+    * Update the specified resource in storage.
+    *
+    * @param \Illuminate\Http\Request $request
+    * @param uid $uid
+    * @return \Illuminate\Http\Response
+    */
+    public function update(Request $request, $uid)
```

Comment:
Validate the request input! 

https://laravel.com/docs/5.0/validation

Reply:
completed